diff --git a/node_modules/ahooks/es/useWebSocket/index.js b/node_modules/ahooks/es/useWebSocket/index.js
index ba2f0d9..7e8a677 100644
--- a/node_modules/ahooks/es/useWebSocket/index.js
+++ b/node_modules/ahooks/es/useWebSocket/index.js
@@ -96,7 +96,7 @@ export default function useWebSocket(socketUrl, options) {
       websocketRef.current.close();
     }
 
-    websocketRef.current = new WebSocket(socketUrl, protocols);
+    websocketRef.current = new (wx ? wx.WebSocket : WebSocket)(socketUrl, protocols);
     setReadyState(ReadyState.Connecting);
 
     websocketRef.current.onerror = function (event) {
